app/models/concerns/hostext/search.rb

```diff
@@ -171,6 +171,19 @@ def search_by_config_group(key, operator, value)
         {:conditions => opts, :include => :hostgroup}
       end
 
+      def cast_facts(key, operator, value)
+        is_int = (value =~ /\A[-+]?\d+\z/ ) or (value.is_a?(Integer))
+        is_pg = ActiveRecord::Base.connection.instance_of? ActiveRecord::ConnectionAdapters::PostgreSQLAdapter
```

Comment:
This causes an error for me when the adapter's not loaded, perhaps linked to the development environment.

I'd suggest doing: `ActiveRecord::Base.connection.adapter_name == 'postgresql'` to match our other similar code.